client/src/components/settings/portfolio.css

```
@@ -0,0 +1,31 @@
+.preview-image {
+  margin-top: 20px;
+  display: block;
+  width: 200px;
+  height: 200px;
+  object-fit: cover;
+  border: 1px solid #ddd;
+  box-shadow: 0 0 5px rgba(0, 0, 0, 0.3);
+  transition: transform 0.2s, box-shadow 0.2s;
+  border-radius: 5px;
+}
+
+.preview-image:hover {
+  transform: scale(1.05);
+  box-shadow: 0 0 10px rgba(0, 0, 0, 0.5);
+}
```

The animation was causing my eyes to get watery when it was light mode, you can try to remake it, but I vote for deleting it, animation and polishing aspects can be done after this PR is merged

```suggestion
  border-radius: 5px;
}
```

---

Hello @Sboonny 
- Done the required changes